app/core/resources/constants/service.py

```diff
@@ -7,17 +7,19 @@
 
 
 # SERVICE CONFIG
-service_section_name = "service"
-NAME = read_config(section=service_section_name, value="name")
-IP = read_config(section=service_section_name, value="ip")
-PORT = read_config(section=service_section_name, value="port")
+service_section_name: str = "service"
+NAME: str = read_config(section=service_section_name, value="name")
+TIMEOUT: int = int(read_config(section=service_section_name, value="timeout"))
+IP: str = read_config(section=service_section_name, value="ip")
+PORT: str = read_config(section=service_section_name, value="port")
```

Comment:
Is it right that the `port` is a string?

Reply:
It's legacy code, When it's used it is converted to int, I will upload a small fix to have int directly here